src/plot_api/plot_api.js

```diff
@@ -1608,6 +1610,8 @@ Plotly.restyle = function restyle(gd, astr, val, traces) {
     var replotAttrs = [
         'zmin', 'zmax', 'zauto',
         'marker.cmin', 'marker.cmax', 'marker.cauto',
+        'line.cmin', 'line.cmax',
+        'marker.line.cmin', 'marker.line.cmax',
```

Comment:
Ha. Good :eyes: . Thanks.

This is good enough for now until https://github.com/plotly/plotly.js/issues/648 is addressed.